frontend/app-development/features/handleMergeConflict/components/HandleMergeConflictAbort.test.tsx

```diff
@@ -13,11 +13,11 @@ const renderHandleMergeConflictAbort = () => {
 
 // find and click the button that opens the popover
 const findAndClickOpenPopoverButton = (user: UserEvent) =>
-  act(() => user.click(
+  user.click(
```

Comment:
All user-events should have keyword await (_await user.click()_), because they are returning a promise. Without await we can have flacky tests. 😃

Reply:
this function returns a promise that is awaited. There is really no need to add extra levels of async/await.